src/components/form/submission.vue

```diff
@@ -16,17 +16,18 @@ except according to the terms contained in the LICENSE file.
 </template>
 
 <script setup>
-import { defineProps, defineOptions, ref, shallowRef } from 'vue';
+import { defineProps, defineOptions, ref, shallowRef, defineAsyncComponent } from 'vue';
 import { useRoute } from 'vue-router';
 import { useI18n } from 'vue-i18n';
-import WebFormRenderer from '../web-form-renderer.vue';
-import EnketoIframe from '../enketo-iframe.vue';
 import Loading from '../loading.vue';
 import useForm from '../../request-data/form';
 import { apiPaths, queryString } from '../../util/request';
 import { noop } from '../../util/util';
+import useProject from '../../request-data/project';
+import { loadAsync } from '../../util/load-async';
 
 const route = useRoute();
+const { project } = useProject();
```

Comment:
`useProject()` has a little bit of a deceptive name. It creates project subresources used in `ProjectShow`, not the actual project resource. Certain resources are created upfront and are not local resources, including `project` and `form`. A resource can't be a local resource if it's referenced in src/routes.js, which these resources are (for page titles, I think).

Instead of calling `useProject()`, you can just grab `project` off `useRequestData()`.

Maybe `useProject()` should be renamed to `useProjectSubresources()`. 🤔